File: Imperium/src/com/imperium/Commands/QueryStatisticsResponse.java
package com.imperium.Commands;

public class QueryStatisticsResponse implements ICommand{
    private CommandType type;
    private int length;
    private String json;

    public CommandType getType() {
        return type;
    }

    public int getLength() {
        return length;
    }

    public void setLength(int length) {
        this.length = length;
    }

    public String getJson() {
        return json;
    }

    public void setJson(String json) {
        this.json = json;
    }

    public QueryStatisticsResponse(){
        type = CommandType.QueryStatisticsResponse;
    }
}

File: Responder/Responder/Commands/QueryStatisticsResponse.cs
namespace Responder.AppConfig
{
    public class QueryStatisticsResponse : ICommand
    {
        #region Properties
        public CommandType Type { get; private set; }
        public int Length { get; set; }
        public string Json { get; set; }
        #endregion

        #region Constructors
        public QueryStatisticsResponse()
        {
            Type = CommandType.QueryStatisticsResponse;
        }
        #endregion
    }
}
